sql/analyzer/indexes.go

```diff
@@ -258,6 +260,43 @@ func getIndexes(
 		}
 
 		return result, nil
+	case *expression.Like:
+		// TODO: maybe more cases to simplify
+		r, ok := e.Right.(*expression.Literal)
+		if !ok {
+			break
+		}
+		val := r.Value()
+		valStr, ok := val.(string)
+		if !ok {
+			break
+		}
+		if len(valStr) == 0 {
+			break
+		}
+		if strings.Count(valStr, "_")-strings.Count(valStr, "\\_") > 0 {
+			break
+		}
+		// if there are no wildcards, this is just a plain equals
+		numWild := strings.Count(valStr, "%") - strings.Count(valStr, "\\%")
```

Comment:
would it be useful to differentiate these two classes with different expression objects or have a wildcard flag on `Like` expressions?